New: OneBlock (#50)
FalixNodes-Software/KB-articles: content/_posts/minecraft/modifications/software/2025-02-06-oneblock.md

3. Navigate to the `oneblock` plugin directory.

4. Open the `config.yml` file. This is the main configuration file where you can customize all available features.

5. Take a look at the options you have there. Some important options you might want to change are:

| Setting                	| Description                                                                                       |
|---------------------------|---------------------------------------------------------------------------------------------------|
| x, y, z                	| The coordinates of where your first OneBlock will be placed                                       |
| Island_for_new_players 	| Whether or not new players get a new island                                                       |
| Rebirth_on_the_island  	| Whether or not the player be tp-ed back to their island after dying                               |
| protection             	| Whether or not players will be prevented from visiting other's islands                            |
| set                    	| The amount of blocks between islands                                                              |

> | СircleMode             	| Whether or not to spawn islands in a circle around the first one                                  |

Reviewer: How did you even manage to put a cyrillic "С" in "СircleMode"?

---
title: Build Your Own Minecraft OneBlock Server
tags: Software
permalink: minecraft/modifications/software/oneblock
description: Create your own OneBlock server, start on a single block and expand your island as you survive with your friends!
keywords:
    - keyword: OneBlock
      matches: ["create", "own"]
author:
    - Twixhunter

icon: "https://cdn.modrinth.com/data/gMMDcC2d/369edc7ea70bc74e8fec7f02e86d57fbce126f6b.jpeg"
mod-name: OneBlock
mod-type: Plugin
mod-url: "https://modrinth.com/plugin/oneblock_bukkit"
---

> ### What is OneBlock?

Reviewer 2: ```suggestion
# What is OneBlock?

```

> OneBlock is a unique Minecraft challenge where you start on a single floating block in the sky. Each time you break the block, a new one is created in its place, offering different materials, chests, and even mobs. As you progress, you go through different levels each with new challenges and resources. The goal is to expand your island and collect resources. OneBlock offers a fun mix of survival, building, and adventure, making it a popular choice for both solo and multiplayer gameplay.

Reviewer 2: ```suggestion
OneBlock is a unique Minecraft challenge where you start on a single floating block in the sky. Each time you break the block, a new one is created in its place, offering different materials, chests, and even mobs. This is done progressively, meaning you will get blocks and items as you progress. You will go through different levels each with new challenges and resources. The goal is to expand your island and collect resources. OneBlock offers a fun mix of survival, building, and adventure, making it a popular choice for both solo and multiplayer gameplay.

```

> ## Installation:

Reviewer 2: ```suggestion
## Playing OneBlock

```

> ## Requirements:

Reviewer 2: ```suggestion
### Requirements

```

> - Make sure your server is running Spigot or one of its forks (Paper, Purpur, etc).

Reviewer 2: ```suggestion
Make sure your server is running Spigot or one of its forks (Paper, Purpur, etc).

```

> ## Installation:

Reviewer 2: ```suggestion
### Installation

```

We already have a guide on how to install OneBlock as well as any other plugin in our [Adding Plugins](/minecraft/modifications/general/adding-plugins) guide.

> ### uploading an empty world.

Reviewer 2: ```suggestion
### Uploading an empty world

```

> Create an empty void world in singleplayer or download one from the internet (like this [Completly empty void](https://www.curseforge.com/minecraft/worlds/completly-empty-void) world) and upload it as explained [here](https://kb.falixnodes.net/minecraft/java/general/world-management).

Reviewer 2: ```suggestion
Create an empty void world in singleplayer or download one from the internet (like this [completly empty void](https://www.curseforge.com/minecraft/worlds/completly-empty-void) world) and upload it as explained [here](https://kb.falixnodes.net/minecraft/java/general/world-management).

```

## Configuring the plugin

1. In the navigation menu, open the "Console & Files" category and navigate to [File Manager](https://client.falixnodes.net/server/filemanager).

2. Open the [Plugins](https://client.falixnodes.net/server/filemanager?dir=/plugins/) folder again.

> 3. Locate and open the [Oneblock](https://client.falixnodes.net/server/filemanager?dir=/plugins/oneblock/) configuration folder.

Reviewer 2: ```suggestion
3. Locate and open the [OneBlock](https://client.falixnodes.net/server/filemanager?dir=/plugins/oneblock/) configuration folder.

```

> 4. Locate and open "config.yml". This is the main configuration file where you will be able to customize any feature you need.

Reviewer 2: ```suggestion
4. Locate and open `config.yml`. This is the main configuration file where you will be able to customize any feature you want to.

```

> 5. Take a look at the options you have here, some important options you might want to change are:

Reviewer 2: ```suggestion
Take a look at the options you have there. Some important options you might want to change are:

```

|------------------------	|--------------------------------------------------------------------------------------------------	|------	|
| X, Y, Z                	| The coordinates where your first OneBlock will be placed.                                         | 6-9  	|
| Island_for_new_players 	| Create separate islands for each player? unchecking this will spawn all players on one island.    | 26   	|
| Rebirth_on_the_island  	| Should the player be tp-ed back to the island after dying in game?                                | 28   	|
| protection             	| Should we prevent players from visiting other islands?                                            | 33   	|
| set                    	| The amount of blocks between islands                                                              | 39   	|
| СircleMode             	| Spawn islands in a circle (true) around the first one or in a line next to each other (false)?    | 41   	|
| Border                 	| Spawn an extending worldborder between islands                                                    | 54   	|

### Creating your first island

> 1. start and join your server

Reviewer 2: ```suggestion
1. Start and join your server.

```

---
title: Build Your Own Minecraft OneBlock Server
tags: Software
permalink: minecraft/modifications/software/oneblock
description: Create your own OneBlock server, start on a single block and expand your island as you survive with your friends!
keywords:
    - keyword: OneBlock
      matches: ["create", "own"]
author:
    - Twixhunter

icon: "https://cdn.modrinth.com/data/gMMDcC2d/369edc7ea70bc74e8fec7f02e86d57fbce126f6b.jpeg"
mod-name: OneBlock
mod-type: Plugin
mod-url: "https://modrinth.com/plugin/oneblock_bukkit"
---

> # What is OneBlock?
> 
> 
> OneBlock is a unique Minecraft challenge where you start on a single floating block in the sky. Each time you break the block, a new one is created in its place, offering different materials, chests, and even mobs. This is done progressively, meaning you will get blocks and items as you progress. You will go through different levels each with new challenges and resources. The goal is to expand your island and collect resources. OneBlock offers a fun mix of survival, building, and adventure, making it a popular choice for both solo and multiplayer gameplay.
> 
> 
> 

Reviewer 3: ```suggestion
## What is OneBlock?

OneBlock is a unique Minecraft challenge where you start on a single floating block in the sky. Each time you break the block, a new one is created in its place, offering different materials, chests, and even mobs. This is done progressively, meaning you will get blocks and items as you progress. You will go through different levels each with new challenges and resources. The goal is to expand your island and collect resources. OneBlock offers a fun mix of survival, building, and adventure, making it a popular choice for both solo and multiplayer gameplay.

```

> # Installation:

Reviewer 3: ```suggestion
## Installation:
```

### Requirements:

Make sure your server is running Spigot or one of its forks (Paper, Purpur, etc).

> ### Installing the plugin

Reviewer 3: ```suggestion
### Installing the plugin

```

We already have a guide on how to install OneBlock as well as any other plugin in our [Adding Plugins](/minecraft/modifications/general/adding-plugins) guide.

### Uploading an empty world

> 
> Create an empty void world in singleplayer or download one from the internet (like this [completly empty void](https://www.curseforge.com/minecraft/worlds/completly-empty-void) world) and upload it as explained [here](https://kb.falixnodes.net/minecraft/java/general/world-management).
> 

Reviewer 3: ```suggestion
Create an empty void world in single-player or download one from the internet (like this [completely empty void](https://www.curseforge.com/minecraft/worlds/completly-empty-void) world) and upload it as explained [here](https://kb.falixnodes.net/minecraft/java/general/world-management).
```

### Configuring the plugin

> 1. In the navigation menu, open the "Console & Files" category and navigate to [File Manager](https://client.falixnodes.net/server/filemanager).
> 
> 2. Open the [Plugins](https://client.falixnodes.net/server/filemanager?dir=/plugins/) folder again.
> 
> 3. Locate and open the [OneBlock](https://client.falixnodes.net/server/filemanager?dir=/plugins/oneblock/) configuration folder.
> 
> 
> 4. Locate and open `config.yml`. This is the main configuration file where you will be able to customize any feature you want to.
> 

Reviewer 3: ```suggestion
1. In the navigation menu, go to the “Console & Files” category and open the [File Manager](https://client.falixnodes.net/server/filemanager).

2. Navigate to the [Plugins](https://client.falixnodes.net/server/filemanager?dir=/plugins/) directory.

3. Find and open the [OneBlock](https://client.falixnodes.net/server/filemanager?dir=/plugins/oneblock/) plugin directory.

4. Open the `config.yml` file. This is the main configuration file where you can customize all available features.
```

> Take a look at the options you have there. Some important options you might want to change are:
> 
> 

Reviewer 3: ```suggestion
5. Take a look at the options you have there. Some important options you might want to change are:

```

> | Setting                	| Description                                                                                       | Line 	|
> |------------------------	|--------------------------------------------------------------------------------------------------	|------	|
> | X, Y, Z                	| The coordinates where your first OneBlock will be placed.                                         | 6-9  	|
> | Island_for_new_players 	| Create separate islands for each player? unchecking this will spawn all players on one island.    | 26   	|
> | Rebirth_on_the_island  	| Should the player be tp-ed back to the island after dying in game?                                | 28   	|
> | protection             	| Should we prevent players from visiting other islands?                                            | 33   	|
> | set                    	| The amount of blocks between islands                                                              | 39   	|
> | СircleMode             	| Spawn islands in a circle (true) around the first one or in a line next to each other (false)?    | 41   	|
> | Border                 	| Spawn an extending worldborder between islands                                                    | 54   	|
> 

Reviewer 3: ```suggestion
| Setting                	| Description                                                                                       | Line 	|
|------------------------	|--------------------------------------------------------------------------------------------------	|------	|
| X, Y, Z                	| The coordinates where your first OneBlock will be placed.                                         | 6-9  	|
| Island_for_new_players 	| Create separate islands for each player? unchecking this will spawn all players on one island.    | 26   	|
| Rebirth_on_the_island  	| Should the player be tp-ed back to the island after dying in game?                                | 28   	|
| protection             	| Should we prevent players from visiting other islands?                                            | 33   	|
| set                    	| The amount of blocks between islands                                                              | 39   	|
| СircleMode             	| Spawn islands in a circle (true) around the first one or in a line next to each other (false)?    | 41   	|
| Border                 	| Spawn an extending world border between islands                                                    | 54   	|

```

An indentation there

### Creating your first island

> 1. Start and join your server.
> 
> 
> 2. Go to the location where you want your first OneBlock to be.
> 
> 3. Run `/ob set` to spawn the first OneBlock.

Reviewer 3: ```suggestion
1. Start and join your server.

2. Go to the location where you want your first OneBlock to be.

3. Run `/ob set` to spawn the first OneBlock.

```
